### PR TITLE
docs: fixed grammar error in vulnerability reporting

### PR DESCRIPTION
**Pull request by abergquist**
Split a run-on sentence into 2 sentences and improved link accessibility.

### CODE REVIEW

File: aio/content/guide/security.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ You can run the <live-example></live-example> in Stackblitz and download the cod
 
 <header>Reporting vulnerabilities</header>
 
-Angular is part of Google [Open Source Software Vulnerability Reward Program](https://bughunters.google.com/about/rules/6521337925468160/google-open-source-software-vulnerability-reward-program-rules), for vulnerabilities in Angular please submit your report [here](https://bughunters.google.com/report).
+Angular is part of Google [Open Source Software Vulnerability Reward Program](https://bughunters.google.com/about/rules/6521337925468160/google-open-source-software-vulnerability-reward-program-rules). [For vulnerabilities in Angular, please submit your report at https://bughunters.google.com](https://bughunters.google.com/report).
```
Comment (josephperrott) [inline aio/content/guide/security.md:16]:
```suggestion
Angular is part of Google [Open Source Software Vulnerability Reward Program](https://bughunters.google.com/about/rules/6521337925468160/google-open-source-software-vulnerability-reward-program-rules). For vulnerabilities in Angular, please submit your report at [https://bughunters.google.com](https://bughunters.google.com/report).
```


We would prefer to only have the url itself as the linked portion.

Comment (pkozlowski-opensource) [reply aio/content/guide/security.md:16]: @josephperrott this PR comes from the ng-conf coaching session and I believe that @jelbourn was sitting down with the person doing the change. I'm going to approve this PR as-is and you both can figure out what is the right change here :-)

**Review by pkozlowski-opensource — APPROVED** [review]
LGTM

Reviewed-for: fw-security

File: aio/content/guide/security.md (hunk 1 above)
Comment (jelbourn) [reply aio/content/guide/security.md:16]: Yeah, this was my recommendation; for a11y, you generally want the link to convey as much context as possible. We could change this, though, to 

```markdown
Angular is part of Google [Open Source Software Vulnerability Reward Program](https://bughunters.google.com/about/rules/6521337925468160/google-open-source-software-vulnerability-reward-program-rules). [For vulnerabilities in Angular, please submit your report to Google Bug Hunters](https://bughunters.google.com/report).
```

Comment (abergquist) [reply aio/content/guide/security.md:16]: Sounds good, Jeremy!

**Review by josephperrott — APPROVED** [review]
LGTM

**Comment by bob-watson** [comment]
@abergquist Please [update this pull request](https://github.com/angular/angular/pull/47326#discussion_r961811283) so we can merge it.
Thanks!

**Review by josephperrott — APPROVED** [review]
LGTM

**Review by josephperrott — APPROVED** [review]
LGTM

Reviwed-for: fw-security

**Review by josephperrott — APPROVED** [review]
LGTM

Reviewed-for: fw-security

**Comment by AndrewKushnir** [comment]
This PR was merged into the repository by commit 29a41031e1fe5e661ed58cc07e02a9c5a100d5a6.

**Pull request closed without merging**